source/chapters/user_interface.rst

```diff
@@ -644,9 +644,14 @@ PFL/Headphone Button
    The headphone buttons of both decks in the mixer
 
 The headphone button is also known as the :term:`pre-fader listen or PFL <PFL>`
-button. Pressing this button allows you to listen and synchronize the track you
-will play next in your headphones before your audience hears the track
-(headphone cueing). You can select more than one PFL button simultaneously.
+button. Pressing this button allows you to listen to and synchronize the track
```

Comment:
`to listen to the next track and synchronizing it with the currently playing track before your audience hears it.`

The `to listen to and` sounds strange to me. The only thing I can come up with to make it sound and read smoother for me is the reordering.